content/api/users/code_snippets/result.api-user-create.sh

```diff
@@ -6,6 +6,7 @@
     "disabled": false,
     "role": null,
     "is_admin": false,
-    "email": "test@datadoghq.com"
+    "email": "test@datadoghq.com",
+    "icon": /path/to/matching/gravatar/icon
```

Comment:
shouldn't it be `"icon": "/path/to/matching/gravatar/icon"` ?

Reply:
Whoops, yes it should be a string, even though it's just a placeholder